migrations-data/updateORC3883.js

```diff
@@ -0,0 +1,369 @@
+
+async function addSubAreas(oldSubAreaList, newORC, newParkName){
+  try {
+      // This needs to makesure the newORC doesnt already have the subareas.
+      // Once that is confirmed create the object
+      // Add info to the park
+      // Create the actual subarea
+      const newSubAreaList = await getSubAreas(newORC);
+      for (const area of oldSubAreaList){
+        if (newSubAreaList.some(obj => obj.sk === area.sk)) {
+          console.log('The subarea ', area.sk, " already exists for ", newORC);
+          continue;
+        }
+        console.log("Creating subAreaName: ", area.subAreaName, " for ", newORC);
+        const newSubAreaOBJ = {
+            activities: area.activities.values,
+            orcs: newORC,
+            managementArea: area.managementArea || " ",
+            section: area.section,
+            region: area.region,
+            bundle: area.bundle,
+            subAreaName: area.subAreaName
+        };
+        let subAreaObj = getValidSubareaObj(newSubAreaOBJ, newParkName);
+        let transactionObj = { TransactItems: [] };
+        transactionObj.TransactItems.push({
+            Update: createUpdateParkWithNewSubAreaObj(
+                subAreaObj.subAreaName,
+                area.sk,
+                subAreaObj.isLegacy,
+                subAreaObj.orcs
+            ),
+        });
+        transactionObj.TransactItems.push({
+            Put: createPutSubAreaObj(subAreaObj, area.sk, newParkName),
+        });
+        await dynamodb.transactWriteItems(transactionObj).promise();
+      }
+    } catch (error) {
+      console.error("Error adding subarea:", error);
+    }
+  }
+    
+async function getConfigs(subArea) {
+  const subAreaID = subArea.sk;
+  const getConfigsQuery = {
+    TableName: TABLE_NAME,
+    KeyConditionExpression: "pk = :pk",
+    ExpressionAttributeValues: {
+      ":pk": { S: `config::${subAreaID}` },
+    },
+  };
+  const config = await runQuery(getConfigsQuery);
+  return config;
+}
+
+async function getActivityRecords(subArea, activity) {
+  const subAreaId = subArea.sk;
+  let activityRecords = [];
+  const getActivitiesQuery = {
+      TableName: TABLE_NAME,
+      KeyConditionExpression: "pk = :pk",
+      ExpressionAttributeValues: {
+          ":pk": { S: `${subAreaId}::${activity}` },
+      },
+  };
+activityRecords = await runQuery(getActivitiesQuery);
+return activityRecords;
+}
+  
+async function getVariances(record, ORC) {
+  const date = record.date;
+  const subAreaId = record.subAreaId;
+  const activity = record.activity;
+  const getVariancesQuery = {
+    TableName: TABLE_NAME,
+    KeyConditionExpression: "pk = :pk AND sk = :sk",
+    ExpressionAttributeValues: {
+      ":pk": { S: `variance::${ORC}::${date}` },
+      ":sk": { S: `${subAreaId}::${activity}` },
+    },
+  };
+  return await runQuery(getVariancesQuery);
+}
+
+const migrationName = 'updateORCS3883.js';
+const oldORC = '0281';
+const newORC = '3883';
+const newParkName = 'Tsutswecw Park';
```

Comment:
This should be `Tsútswecw Park` based on what is in the data register.

Reply:
Addressed in commit:[9792](https://github.com/bcgov/bcparks-ar-api/pull/306/commits/9792e47e834642005d0b2e4277a0b790e8b9fddb)